modules/viewing-the-service-logs-cli.adoc

```diff
@@ -0,0 +1,143 @@
+// Module included in the following assemblies:
```

Comment:
**Note**: The includes for this module file are commented out in the associated assemblies, so that they do not show in the live build.